example/raylib-tmx-example.c

```diff
@@ -27,7 +27,7 @@ int main(int argc, char *argv[]) {
     InitWindow(screenWidth, screenHeight, "[raylib-tmx] example");
     SetTargetFPS(60);
 
-    tmx_map* map = LoadTMX("resources/desert.tmx");
+    tmx_map* map = LoadTMX(argv[1]);
```

Comment:
Thanks! Unrelated change?
``` suggestion
    tmx_map* map = LoadTMX("resources/desert.tmx");
```

Reply:
I wanted to test different tmx files and passing via command line was the easiest way, but I don't mind reverting it back

Reply:
Good idea... Perhaps we check if a command line argument is passed, and use it if it's there. Otherwise, default to resources/desert.tmx :wink:

Reply:
Sure, that is great, but let's make a new PR then and revert this one for now